src/collections/components/StoryForm/StoryForm.tsx

```diff
@@ -227,6 +292,24 @@ export const StoryForm: React.FC<StoryFormProps & SharedFormButtonsProps> = (
     formik.setFieldValue('excerpt', applyCurlyQuotes(formik.values.excerpt));
   };
 
+  const showTextSwitchLink = (
+    linkDescriptionText: string,
+    textToInsert: string,
+    newDescriptionState: DescriptionTextStates
+  ) => (
+    <Tooltip title={textToInsert} arrow>
+      <StyledLink
+        onClick={() => {
+          formik.setFieldValue('excerpt', textToInsert);
+          setDescriptionState(newDescriptionState);
+        }}
+        sx={{ textDecoration: 'none' }}
+      >
+        {linkDescriptionText}
+      </StyledLink>
+    </Tooltip>
+  );
```

Comment:
I would move this into a separate helper / shared component since it's returning jsx and we try to keep just one jsx component per file.

Reply:
ok, I split it up